### PR TITLE
Update contexts.md

### PR DESCRIPTION
**Pull request by krzysztof-pawlicki-nemo**
Updated inputs context description with missing type `choice` information and provided more precise information about standard properties with the link to workflow dispatch event trigger for reference.

### Why:

Fixes misleading or missing information in the doc.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Inputs documentation.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Review by AliTahirGujjar — COMMENTED** [review]
zainmotorsofficial

**Review by AliTahirGujjar — COMMENTED** [review]
zainmotorsofficial

**Review by AliTahirGujjar — COMMENTED** [review]
Approove

**Comment by cmwilson21** [comment]
@krzysztof-pawlicki-nemo Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/actions/learn-github-actions/contexts.md [hunk 1]
```diff
@@ -853,14 +853,14 @@ jobs:
...
 
-There are no standard properties in the `inputs` context, only those which are defined in the workflow file.
+The standard properties in the inputs are defined only for [`workflow_dispatch` event](/actions/learn-github-actions/events-that-trigger-workflows#workflow_dispatch).
```
Comment (skedwards88) [inline content/actions/learn-github-actions/contexts.md:856]: I believe this sentence was trying to convey that the property names are dependent on the workflow. How about something like:
```suggestion
The properties in the `inputs` context are defined in the workflow file. They are only available in a [reusable workflow](/actions/learn-github-actions/reusing-workflows){% ifversion actions-unified-inputs %} or in a workflow triggered by the [`workflow_dispatch` event](/actions/learn-github-actions/events-that-trigger-workflows#workflow_dispatch){% endif %}
```

**Review by skedwards88 — COMMENTED** [review]
Thanks for this update! I made one wording suggestion that I'll go ahead and incorporate, then we'll get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**